truss/types.py

```diff
@@ -71,3 +71,26 @@ def from_dict(patch_details: Dict) -> "PatchDetails":
                 Patch.from_dict(patch_op) for patch_op in patch_details["patch_ops"]
             ],
         )
+
+
+@dataclass
```

Comment:
is it possible to use pydantic over dataclasses to benefit from the FastAPI typing when it comes to sending to the control server?